index.html

```diff
@@ -262,7 +319,7 @@ <h2 class="recent-contributors-head">Recent Contributors</h2>
         </div>
       </div>
 
-      <h2 class="contributors-head">Contributors</h2>
+      <h2 class="contributors-head" id="contributer">Contributors</h2>
```

Comment:
typo in the div name @YogeshSharma01

Reply:
Yeah Thanks, @govindgoel I have fixed the typo